openio-sds/openio-sds.spec

```diff
@@ -140,10 +140,6 @@ Requires:       python-gunicorn    >= 19.4.5
 Requires:       python-flask,python-eventlet,python-zmq,python-redis,python-requests >= 2.6.0,python-plyvel,PyYAML,python-futures
```

Comment:
- `python-flask` seems no more used
- `python-requests` is only used for Backblaze connector. It could be listed on a separate line with a comment explaining that.